chart/templates/NOTES.txt

```diff
@@ -176,6 +176,14 @@ DEPRECATION WARNING:
 
 {{- if not (or .Values.webserverSecretKey .Values.webserverSecretKeySecretName) }}
 
+{{- if .Values.securityContext }}
+
+ DEPRECATION WARNING:
+    `securityContext` has been renamed to `securityContexts`, to be enabled on container and pod level.
```

Comment:
We also need the corresponding update to the `airflowSecurityContext` helper to use the new config also.

Reply:
tried to change the helper accordingly